packages/core/src/components/viewer-grid/viewer-grid.global.scss

```diff
@@ -113,23 +113,6 @@ dso-viewer-grid {
     }
   }
 
-  [slot="overlay"] {
```

Comment:
Dit is bijvangst, deze styling had nooit in de `.global` stylesheet gemogen: Het element wat hier gestijld wordt is niet van `dso-viewer-grid`. Ik heb de styling verplaatst naar `.overlay ::before` in de Shadow root